src/genericSchemeRegistry/schemes/BalancerPoolManager.json

```diff
@@ -0,0 +1,341 @@
+{
+  "name": "BalancerPoolManager",
+  "addresses": {
+    "main": [
+      ""
+    ],
+    "kovan": [
+      "0x5086d91858a5aac4a770212720b297f38e0dd403"
+    ],
+    "private": [
+      ""
+    ]
+  },
+  "actions": [
+    {
+      "id": "setPublicSwap",
+      "label": "Set PublicSwap parameter to true/false",
+      "description": "Pause/unpauses the pool",
+      "notes": "https://github.com/PrimeDAO/monorepo/blob/master/packages/contracts/contracts/schemes/BalancerProxy.sol",
+      "fields": [
+        {
+          "label": "Set public swap to",
+          "name": "public swap",
+          "placeholder": "Boolean (true)"
+        }
+      ],
+      "abi": {
+        "constant": false,
+        "inputs": [
+          {
+            "internalType": "bool",
+            "name": "publicSwap",
+            "type": "bool"
+          }
+        ],
+        "name": "setPublicSwap",
+        "outputs": [],
+        "payable": false,
+        "stateMutability": "nonpayable",
+        "type": "function"
+    }
+    },
+    {
+      "id": "setSwapFee",
+      "label": "Set Swap Fee",
+      "description": "Update the Swap Fee",
+      "notes": "https://github.com/PrimeDAO/monorepo/blob/master/packages/contracts/contracts/schemes/BalancerProxy.sol",
+      "fields": [
+        {
+          "decimals": 18,
+          "label": "The swap fee is expressed as a percentage.",
+          "name": "amount",
+          "unit": "%",
+          "placeholder": "Swap fee (1.5)"
+        }
+      ],
+      "abi": {
+          "constant": false,
+          "inputs": [
+            {
+              "internalType": "uint256",
+              "name": "swapFee",
+              "type": "uint256"
+            }
+          ],
+          "name": "setSwapFee",
+          "outputs": [],
+          "payable": false,
+          "stateMutability": "nonpayable",
+          "type": "function"
+          }
+    },
+    {
+      "id": "commitAddToken",
+      "label": "Commit add token to the balancer pool",
+      "description": "First step in adding a new token to the balancer pool",
+      "notes": "https://github.com/PrimeDAO/monorepo/blob/master/packages/contracts/contracts/schemes/BalancerProxy.sol",
+      "fields": [
+        {
+          "label": "Token address",
+          "name": "token",
+          "placeholder": "Address (0x0000…)"
+        },
+        {
+          "decimals": 18,
+          "label": "Balance of the new Token to add to the pool.",
```

Comment:
We should remove the period (".") from all labels and placeholders and be consistent with all capitalization in all labels and placeholders (only capitalize the first letter).Note I have not corrected them all in my comments, so we should review them all.

"Balance of the new token"